frontend/src/metabase/dashboard/actions.js

```diff
@@ -707,15 +705,10 @@ export const fetchDashboard = createThunkAction(FETCH_DASHBOARD, function(
     }
 
     const parameterValuesById = preserveParameters
-      ? getParameterValuesByIdFromPreviousValues(
```

Comment:
Removed this util function because it didn't actually do anything to the given `parameterValues` value besides instantiate a new array